packages/cosmos_ui_components/lib/components/cosmos_app_bar.dart

```diff
@@ -30,17 +30,18 @@ class CosmosAppBar extends StatelessWidget implements PreferredSizeWidget {
             ],
           ),
           const SizedBox(height: 20),
```

Comment:
you have to remove the SizedBox aswell, if the title is null, otherwise it will still overflow

Reply:
I removed the sized box. Still it's giving an overflow for 11 pixels. I think it's due to `SafeArea`